docs/reST/ref/display.rst

```diff
@@ -571,4 +571,50 @@ required).
 
    .. ## pygame.display.get_window_size ##
 
+.. function:: get_screensaver_enabled
+
+   | :sl:`Return whether SDL is hinted to enable or disable the screensaver.`
+   | :sg:`get_screensaver_enabled() -> bool`
+
+   Return whether SDL is hinted to enable or disable the screensaver.
+   This function is only available with SDL2.  Under SDL1, this
+   function will raise an exception.
+
+   .. note:: The reliability of this function is subject to platform
+             support.  Please see
+             :func:`pygame.display.set_screensaver_enabled()` for
+             caveats with screensaver support.
+
+   .. versionadded:: 2.0
+
+   .. ## pygame.display.get_screensaver_enabled ##
+
+.. function:: set_screensaver_enabled
+
+   | :sl:`Change request state for whether screensaver may run`
+   | :sg:`set_screensaver_enabled() -> None`
+
+   Change the SDL hint for whether screensavers should be enabled or
+   disabled.  This function is only available with SDL2.  Under SDL1,
+   this function will raise an exception.
+
+   If the screen has been disabled due to this function (or by default
+   in SDL), it will automatically be re-enabled when
+   :func:``pygame.quit()` is called.
+
+   It is possible to influence the default value under SDL 2.0.2 or
+   later via the environment variable ``SDL_HINT_VIDEO_ALLOW_SCREENSAVER``,
+   which can be set to either ``0`` (disable) or ``1`` (enable).
+
```

Comment:
Could add a note here that `SDL_HINT_VIDEO_ALLOW_SCREENSAVER` defaults to False, so by default screen savers are disabled while pygame applications are running.